packages/lesswrong/themes/lesswrongTheme.js

```diff
@@ -20,6 +21,7 @@ const sansSerifStack = [
 ].join(',')
 
 const serifStack = [
+  'Valkyrie',
   'warnock-pro',
```

Comment:
This has users downloading both fonts, even though Warnock is only used as a fallback for glyphs that are missing from Valkyrie (which Warnock isn't likely to have either).